inst/odin/carehomes.R

```diff
@@ -3,26 +3,48 @@
 ## j for the progression (not exponential latent and infectious period)
 ## k for the infectivity group (for I) or vacc. group (for S)
 
-## Number of classes (age transmissibility & vaccination)
+## Number of classes (age & vaccination)
 
 ## Number of "groups", being the age classes, Carehome workers and
 ## Carehome residents. This will be 19 in all but experimental uses.
 n_age_groups <- user()
 n_groups <- user()
-n_trans_classes <- user(1)
 
 ## Definition of the time-step and output as "time"
 dt <- user()
 initial(time) <- 0
 update(time) <- (step + 1) * dt
+## output number of individuals vaccinated by age and clinical stage
+## vaccinated S
+initial(cumul_n_S_vaccinated[]) <- 0
+update(cumul_n_S_vaccinated[]) <- cumul_n_S_vaccinated[i] +
+  n_S_next_vacc_class[i, 1] + n_SE_next_vacc_class[i, 1]
+dim(cumul_n_S_vaccinated) <- n_groups
+## vaccinated E
+initial(cumul_n_E_vaccinated[, ]) <- 0
+update(cumul_n_E_vaccinated[, ]) <- cumul_n_E_vaccinated[i, j] +
```

Comment:
it's not obvious to me why there are *two* things being added here at each step for each class. Could you comment as to what is going on there please? (i.e., why is E accumulating both E and EE). This might be obvious from the epi?

Reply:
I've added a comment that hopefully helps explain this. We may be able to tidy some of this up later.